src/test/play-27/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/changeList.scala.html

```diff
@@ -46,13 +46,13 @@ <h1 class="govuk-heading-xl">Change Elizabeth Keen’s details</h1>
             )
           )
         ))
-      ),
+      ), 
       SummaryListRow(
         key = Key(
           content = Text("Does Elizabeth Kean have a National Insurance number?")
         ),
         value = Value(
-          content = HtmlContent("Yes")
+          content = HtmlContent(Yes)
```

Comment:
Does this work? I thought the `Yes` needed to the wrapped as a string `"Yes"`

Reply:
Yes, it appears the play-27 and play-28 directories are not actually compiled, which is a bit worrying.